### DISCUSSION TITLE
Animaux omnivores-hippopotame

### DISCUSSION THREAD

**Discussion by LeHoublonPensant** [Crownicles/Crownicles#881] [Suggestions fermées]
L'hippopotame est omnivore, pourtant il est seulement herbivore dans le jeu.◀à changer

**Comment by MF4db** [comment]
Ça existe déjà, il y a quelques familiers déjà omnivores, après je ne sais si les régimes de tous les familiers sont corrects.

**Reply by LeHoublonPensant** [reply]
Dans ce cas ma nouvelle suggestion : rendre les hippos omnivores

**Comment by Feiryn** [comment]
D'après toutes les sources que je trouve les hippopotames sont herbivores

**Reply by LeHoublonPensant** [reply]
> D'après toutes les sources que je trouve les hippopotames sont herbivores

Normal, c'est une découverte récente : demande à romain 22222 !!

**Reply by LeHoublonPensant** [reply]
Et encore y'a des articles de 2015 qui en parlent 🙄

**Reply by angels-hub** [reply]
> Cependant, les hippopotames restent avant tout des herbivores dont le comportement carnivore serait induit par des facteurs climatiques stressants tels que la sécheresse ou l'apparition de ressources surabondantes (cadavres d'hippopotames infectés par l'anthrax, noyade massive de gnous).
https://www.sciencesetavenir.fr/animaux/biodiversite/les-hippopotames-ces-herbivores-carnivores_103496

**Reply by LeHoublonPensant** [reply]
Bah n'empêche qu'ils peuvent manger de la viande, et le joueur les nourrir avec...

**Reply by LeHoublonPensant** [reply]
J'avais lu le même article 😄

**Reply by Feiryn** [reply]
Oui donc c'est très rare, ça n'en fait pas son régime alimentaire principal. N'importe quel animal peut manger n'importe quoi mais ça n'en fait pas pour autant des omnivores.

**Reply by LeHoublonPensant** [reply]
Oui BastLast a répondu de toute façon, le sujet est clos 😉

**Reply by angels-hub** [reply]
c'est moi qui avait défini les régimes alimentaires des familiers à l'époque avec @voltou91 et on était parti d'une méthode simple : chercher sur internet le regime alimentaire et prendre le régime alimentaire domminant sauf pour les familiers fictifs

**Comment by BastLast** [comment]
c'est moi qui décide c'est non Bonne journée

**Reply by LeHoublonPensant** [reply]
> c'est moi qui décide c'est non Bonne journée

Ok parfait !! Voilà qui tranche la question !!
